web/modules/custom/joinup_core/src/EventSubscriber/RefreshCachedFieldsEventSubscriber.php

```diff
@@ -355,4 +335,37 @@ protected function getPiwikSettings($bundle) {
     return FALSE;
   }
 
+  /**
+   * Builds a list of parameters for a query.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity with the expired field.
+   * @param array $parameters
+   *   A list of extra parameters to pass to the array of parameters.
+   *
+   * @return array
+   *   A list of parameters.
+   */
+  protected function getSubqueryParams(EntityInterface $entity, array $parameters = []): array {
```

Comment:
Hmm what is this? Do you have any documentation for it? It looks like this is a feature from Piwik, meaning that it should better belong in the library or contrib module instead of in custom code. At least then this code is reusable.